.github/workflows/build.yaml

```diff
@@ -28,7 +28,7 @@ concurrency:
 jobs:
   cpp-build:
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-build.yaml@branch-24.04
+    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-build.yaml@python-3.11
```

Comment:
Can the workflow branch changes be reverted now that https://github.com/rapidsai/shared-workflows/pull/176 is merged?

Reply:
grrr I swear I thought I already did that. Yep, just pushed that change and did that on the `cudf` one as well.